packages/terra-markdown/src/terra-dev-site/test/markdown/Markdown.test.jsx

```diff
@@ -1,11 +1,9 @@
 import React from 'react';
 import Markdown from '../../../../lib/Markdown';
-
-const markdownSrc = `### Test
-> This is a quote.`;
```

Comment:
I'm going to add this back to the test, I realized I wasn't covering this any more.